### DISCUSSION TITLE
how to Build and use videochat

### DISCUSSION THREAD

**Discussion by FxamarAboali** [nkonev/videochat#6] [General]
hi

I am started to try Build and use videochat  app but I can't do anything 

can anyone help me how to do it 
video learn how to Build and use videochat .

thank you!

**Comment by nkonev** [comment]
To build this project you can use makefiles.

To understand which goals of make and which software and versions you need - you can study https://github.com/nkonev/videochat/blob/master/.github/workflows/ci.yml
